.prettierignore

```diff
@@ -1,7 +1,15 @@
 *.d.ts
+.codesandbox
+.git
+.github
+.storybook
```

Comment:
How come all of these got added? I  think we should still include all of these  except for the .git folder.

Reply:
I didn't realize these were configurations we added. I assumed these were autogenerated similar to `.cache` on clayui.com